modules/juce_audio_processors/format_types/juce_VST3PluginFormat.cpp

```diff
@@ -1103,6 +1101,30 @@ struct VST3ModuleHandle  : public ReferenceCountedObject
         return false;
     }
 
+    Result findName(IPluginFactory* factory, const PluginDescription& description)
+    {
+        if (factory == nullptr)
+            return Result::fail(L"Invalid Parameter");

```

Comment:
> `L""`

Why are you adding `L`? It's not necessary at all.

Reply:
Sorry, too used to code for Windows Unicode. will remove it

Reply:
done